README.md

```diff
@@ -1,9 +1,53 @@
-
 # Telegraf
 
-![tiger](assets/TelegrafTiger.png "tiger")
-
-[![Contribute](https://img.shields.io/badge/Contribute%20To%20Telegraf-orange.svg?logo=influx&style=for-the-badge)](https://github.com/influxdata/telegraf/blob/master/CONTRIBUTING.md) [![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=for-the-badge)](https://www.influxdata.com/slack) [![Circle CI](https://circleci.com/gh/influxdata/telegraf.svg?style=svg)](https://circleci.com/gh/influxdata/telegraf) [![GoDoc](https://godoc.org/github.com/influxdata/telegraf?status.svg)](https://godoc.org/github.com/influxdata/telegraf) [![Docker pulls](https://img.shields.io/docker/pulls/library/telegraf.svg)](https://hub.docker.com/_/telegraf/) [![Go Report Card](https://goreportcard.com/badge/github.com/influxdata/telegraf)](https://goreportcard.com/report/github.com/influxdata/telegraf)
+<div align="center">
+  <img width="600" src="assets/telegraf-logo.png" alt="Telegraf Logo">
+</div>
```

Comment:
Having two logos at the top of the README.md seems a bit much, I prefer we keep the tiger at the top and maybe move the `telegraf` picture elsewhere or remove it. It doesn't look great with a dark theme.